src/installer/installer.go

```diff
@@ -53,6 +53,7 @@ const (
 	nodeImagePullService         = "node-image-pull.service"
 	nodeImageOverlayService      = "node-image-overlay.service"
 	openshiftClientBin           = "/usr/bin/oc"
+	registryDataDir              = "/mnt/agentdata"
```

Comment:
(not for this PR) but I'm wondering if in future we could also review the Appliance folders naming, for something more inline with the [EP](https://github.com/openshift/enhancements/pull/1821) concepts (ie, instead of `agentdata` we could use something like `ocp-release-bundle-data`)